### PR TITLE
Rebranded api to force trace drop/keep

### PR DESCRIPTION
**Pull request by labbati**
### What does this PR do?
1. Priority Sampling is enabled by default for all languages 
2. APM is now called APM & Distributed Tracing 
3. Introduced Force Tracing as an overriding capability to manually keep/drop a trace and has been moved to Guide section. -- Priority sampling removed from guides and sampling and storage section. Priority sampling env variable removed. 

### Preview link
[Preview Link](https://docs-staging.datadoghq.com/apm/rebrand-priority-sampling/tracing/guide/)

###

### CODE REVIEW

File: content/en/tracing/guide/distributed_tracing.md [hunk 1]
```diff
@@ -250,31 +250,26 @@ For a more detailed explanations of sampling and priority sampling, check the [s
...
 
 public class MyClass {
     @Trace
     public static void myMethod() {
         // grab the active span out of the traced method
         MutableSpan ddspan = (MutableSpan) GlobalTracer.get().activeSpan();
+
         // ask the sampler to keep the current trace
-        ddspan.setSamplingPriority(PrioritySampling.USER_KEEP);
+        ddspan.setTag(ForcedTracing.manual_KEEP);
```
Comment (brettlangdon) [inline content/en/tracing/guide/distributed_tracing.md]: `MANUAL_KEEP`/`MANUAL_DROP`?

Assuming these have to be all caps?

**Comment by brettlangdon** [comment]
I updated this PR to include changes for Ruby and Python. I modeled them after the PHP section.

File: content/en/tracing/guide/distributed_tracing.md (hunk 1 above)
Comment (palazzem) [reply content/en/tracing/guide/distributed_tracing.md]: Yes, @labbati can you make the change?

Comment (labbati) [reply content/en/tracing/guide/distributed_tracing.md]: A couple of other things to fix. Doing it right now thanks!

File: content/en/tracing/guide/distributed_tracing.md [hunk 2]
```diff
@@ -427,6 +384,25 @@ Priority sampling is enabled by default. The default sampler automatically assig
 
 Priority sampling is enabled by default.
 
+
+Manually set trace priority:
+
+```php
+
+$tracer = \OpenTracing\GLobalTracer::get();
```
Comment (cgilmour) [inline content/en/tracing/guide/distributed_tracing.md]: `GLobalTracer` -> `GlobalTracer`

Comment (labbati) [reply content/en/tracing/guide/distributed_tracing.md]: 👍 thanks good catch! :)

File: content/en/tracing/guide/force_tracing.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+
+Manually keep or drop a trace:
+```cs
+using Datadog.Trace;
+
+// Access the active scope through the global tracer (can return null)
+var scope = Tracer.Instance.ActiveScope;
+
+// Always keep this trace
+scope.Span.SetTag(Tags.ManualKeep, true);
```
Comment (lucaspimentel) [inline content/en/tracing/guide/force_tracing.md]: Currently the method signature is `SetTag(string ,string)`, so you can't pass a bool value.

```suggestion
scope.Span.SetTag(Tags.ManualKeep, "true");
```

File: content/en/tracing/guide/force_tracing.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+
+// Always drop this trace
+scope.Span.SetTag(Tags.ManualDrop, true);
```
Comment (lucaspimentel) [inline content/en/tracing/guide/force_tracing.md]: Same as above.

```suggestion
scope.Span.SetTag(Tags.ManualDrop, "true");
```

File: content/en/tracing/_index.md [hunk 5]
```diff
@@ -14,7 +14,16 @@ disable_toc: true
 
 Datadog APM provides you with deep insight into your application's performance-from automatically generated dashboards monitoring key metrics, such as request volume and latency, to detailed traces of individual requests-side by side with your logs and infrastructure monitoring.
 
-Datadog APM is offered as an upgrade to the Pro and Enterprise plans. A free 14-day trial is available. Registered users can visit the [APM page of the Datadog application][1] to get started.
+## Distributed Tracing
+
+Distributed tracing allows you to propagate a single trace across multiple services and hosts, so you can see performance end-to-end. Linking is implemented by injecting Datadog Metadata into the request headers.
```
Comment (cswatt) [inline content/en/tracing/_index.md]:
```suggestion
Distributed tracing allows you to propagate a single trace across multiple services and hosts, so you can see performance end-to-end. Linking is implemented by injecting Datadog metadata into the request headers.
```

File: content/en/tracing/guide/force_tracing.md [hunk 6]
```diff
@@ -0,0 +1,205 @@
+---
+title: Force Tracing
+kind: documentation
+aliases:
+  - /tracing/guide/force_tracing
+---
+
+
+APM enables priority sampling by default to allow traces between two Datadog endpoints to be sampled together. This prevents trace sampling from removing segments of a distributed trace (i.e. ensures completeness) and helps removing unimportant ones. You can override this functionality to force keep a trace (critical transaction, debug mode, etc.) or force drop a trace (health checks, static assets, etc) by the agent and the server using forced tracing.
```
Comment (cswatt) [inline content/en/tracing/guide/force_tracing.md]:
```suggestion
APM enables priority sampling by default to allow traces between two Datadog endpoints to be sampled together. This prevents trace sampling from removing segments of a distributed trace (i.e. ensures completeness) and helps to remove unimportant ones. You can override this functionality to force-keep a trace (critical transaction, debug mode, etc.) or forc- drop a trace (health checks, static assets, etc.) by the agent and the server using forced tracing.
```

File: content/en/tracing/guide/force_tracing.md [hunk 7]
```diff
@@ -0,0 +1,205 @@
...
+
+    // Always drop this trace
+    span.SetTag(Tags.ManualDrop, "true");
+}
+```
+
+{{% /tab %}}
+{{% tab "PHP" %}}
+
+Anywhere in your code you can easily access the currently active span and force the current trace to be kept or dropped.
```
Comment (cswatt) [inline content/en/tracing/guide/force_tracing.md]:
```suggestion
From anywhere in your code, you can access the currently active span and force the current trace to be kept or dropped.
```

File: content/en/tracing/guide/force_tracing.md (hunk 6 above)
Comment (priyanshi-gupta) [reply content/en/tracing/guide/force_tracing.md]: Forc -> Force* in the review

File: content/en/tracing/guide/force_tracing.md [hunk 8]
```diff
@@ -0,0 +1,205 @@
...
+```python
+from ddtrace import tracer
+from ddtrace.constants import MANUAL_DROP_KEY, MANUAL_KEEP_KEY
+
+@tracer.wrap()
+def handler():
+    span = tracer.current_span()
+
+    # Manually choose to drop healthcheck traces
+    if is_healthcheck():
```
Comment (priyanshi-gupta) [inline content/en/tracing/guide/force_tracing.md]: Do we need these checks - is_healthcheck()? If it is only as a sample and not a method DD provides - I will suggest removing it as healthcheck is one example, there can be other use cases as well.

Comment (brettlangdon) [reply content/en/tracing/guide/force_tracing.md]: sure, we don't need `is_healthcheck()` here, I was just trying to give a realistic example of when you would manually choose to drop a trace.

Same with `is_error()`.

The goal was to not encourage people to manually keep or drop everything.

**Comment by ruthnaebeck** [comment]
@labbati @brettlangdon @priyanshi-gupta - Is this something that you still plan to merge?

**Pull request merged**